esphome/components/modbus_controller/__init__.py

```diff
@@ -254,6 +256,8 @@ async def add_modbus_base_properties(
 async def to_code(config):
     var = cg.new_Pvariable(config[CONF_ID])
     cg.add(var.set_command_throttle(config[CONF_COMMAND_THROTTLE]))
+    if CONF_MAX_CMD_RETRIES in config:
+        cg.add(var.set_max_cmd_retries(config[CONF_MAX_CMD_RETRIES]))
```

Comment:
```suggestion
    cg.add(var.set_max_cmd_retries(config[CONF_MAX_CMD_RETRIES]))
```
It will always be set since a default is provided.

Reply:
Got it wrong, will fix.